frontend/catalyst/compiler.py

```diff
@@ -63,6 +65,7 @@ class CompileOptions:
     pipelines: Optional[List[Any]] = None
     autograph: Optional[bool] = False
     lower_to_llvm: Optional[bool] = True
+    abstracted_axes: Optional[Iterable[Iterable[str]] | Dict[int, str]] = None
```

Comment:
Based on the codefactor warnings, do Dict and Iterable need to be imported from typing?

Reply:
Yes, it was an error. Should be fixed already.